package/yast2-network.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Apr 27 10:44:05 UTC 2021 - Michal Filka <mfilka@suse.com>
+
+- Dropped LanItems, replaced by network-ng
```

Comment:
Miss bug reference just use this one https://bugzilla.suse.com/show_bug.cgi?id=1185338

```suggestion
- Dropped LanItems, replaced by network-ng (bsc#1185338)
```